src/redist/targets/GenerateBundledVersions.targets

```diff
@@ -206,8 +212,9 @@ Copyright (c) .NET Foundation. All rights reserved.
     @(ImplicitPackageVariable->'<ImplicitPackageReferenceVersion Include="%(Identity)" TargetFrameworkVersion="%(TargetFrameworkVersion)" DefaultVersion="%(DefaultVersion)" LatestVersion="%(LatestVersion)"/>', '
     ')
 
+    <!-- .NET 6.0 -->
     <KnownFrameworkReference Include="Microsoft.NETCore.App"
-                              TargetFramework="netcoreapp5.0"
+                              TargetFramework="netcoreapp6.0"
```

Comment:
Doesn't `net6.0` work? I assumed that we only kept support for `netcoreapp5.0` because there wasn't a need to remove it but not continue supporting later netcoreapp tfms.